### ISSUE TITLE
Research Citation Style Language patch

### ISSUE THREAD

**Issue by alawvt** [VTUL/vtechworks#706]
Consider installing Atmire's [Citation Style Language patch](https://github.com/atmire/CSL63) for DSpace 6. 

Atmire announcement: [University of Bordeaux releases Citation Styles for DSpace](https://www.atmire.com/articles/detail/university-of-bordeaux-releases-citation-styles-for-dspace)

[Oskar-Bordeaux Open Science & Knowledge Archive](https://oskar-bordeaux.fr/?locale-attribute=en), the DSpace 6.3 XMLUI repository of the University of Bordeaux

## Sample item
"FDI and macroeconomic volatility: a close-up on the source countries"
- https://oskar-bordeaux.fr/handle/20.500.12278/3932?locale-attribute=en
- https://oskar-bordeaux.fr/handle/20.500.12278/1622?show=full

sample citation export (Vancouver, text)
>1. ATCHER J, NAGAI A, MAURIZOT V, TANATANI A, HUC I. Aromatic beta-sheet foldamers based on tertiary squaramides [Internet]. Vol. 55, Chemical Communications. 2019. p. 10392–5. Available from: https://oskar-bordeaux.fr/handle/20.500.12278/3932

sample citation export (Turabian, text)
>ATCHER, Joan, Aki NAGAI, Victor MAURIZOT, Aya TANATANI, and Ivan HUC. “Aromatic Beta-Sheet Foldamers Based on Tertiary Squaramides.” Journal article. Chemical Communications, -1, 2019. https://oskar-bordeaux.fr/handle/20.500.12278/3932.

sample citation export (Nature, text)
>1.ATCHER, J., NAGAI, A., MAURIZOT, V., TANATANI, A. & HUC, I. Aromatic beta-sheet foldamers based on tertiary squaramides. Chemical Communications vol. 55 10392–10395 (2019).

sample citation export (IEEE, text)
>[1]J. ATCHER, A. NAGAI, V. MAURIZOT, A. TANATANI, and I. HUC, “Aromatic beta-sheet foldamers based on tertiary squaramides,” Chemical Communications, vol. 55, no. 70. pp. 10392–10395, -1-2019 [Online]. Available: https://oskar-bordeaux.fr/handle/20.500.12278/3932

sample citation export (Harvard, text)
>ATCHER, J., NAGAI, A., MAURIZOT, V., TANATANI, A. and HUC, I., 2019. Aromatic beta-sheet foldamers based on tertiary squaramides. Chemical Communications, <https://oskar-bordeaux.fr/handle/20.500.12278/3932>.

## Notes 
- record does not contain a citation field
- record does contain a COinS field (it may not be relevant but it is interesting)
- from hal.description.error, it looks like it was a SWORD submission

## References
- [Citation Style Language](https://citationstyles.org)

**Comment by alawvt** [comment]
I installed the patch with our current theme, `<theme name="Mirage 2 Theme VT colors" regex=".*" path="vtmirage2/" />`, instead of the default, `<theme name="Atmire Mirage Theme" regex=".*" path="Mirage/" />`. The menu on the item displays but is not complete and the links do not go anywhere. Also, I don't see a button at the top of the page.
<img width="1000" alt="CitationStyleLanguagePatchVT_theme" src="https://user-images.githubusercontent.com/13037168/128032931-29bcb1fc-6ba8-4a65-ae96-36514f1792fd.png">

The Citation format page, http://192.168.60.4/citations, linked in My Account seems ok.

I will repeat this with the default theme.

**Comment by alawvt** [comment]
I ran the patch with the default Mirage 2 theme. It built and I added APA.csl. However, when I selected that style and any format, it produced the error,
<img width="1040" alt="Citation_Style_Language_Patch_error" src="https://user-images.githubusercontent.com/13037168/128058381-d64cbb24-ea5a-47df-9557-c64be91b4a5d.png">
Since this patch is based on XMLUI, which will not be used in DSpace 7, I will halt this testing.
